packages/x-components/src/__stubs__/suggestions-stubs.factory.ts

```diff
@@ -1,43 +1,37 @@
-import { ModelNameType, Suggestion } from '@empathyco/x-types';
-import { createFacetWithFilter } from './facets-stubs.factory';
+import { Suggestion } from '@empathyco/x-types';
 
 /**
- * Function to create suggestions stub with the modelName passes as parameter.
+ * Creates {@link @empathyco/x-types#Suggestion | suggestions} stub.
  *
- * @param modelName - Model name for each suggestion.
- * @returns Array of suggestion stub.
+ * @param amount - Number of stubbed suggestions to create.
+ *
+ * @returns Array of suggestions stub.
+ *
+ * @internal
+ */
+export function getSuggestionStub(amount = 3): Suggestion[] {
```

Comment:
This change from `getSuggestionsStub` to `getSuggestionStub` breaks some tests, please take a look.